dpdispatcher/contexts/openapi_context.py

```
@@ -23,6 +23,36 @@
 
 
+def unzip_file(zip_file, out_dir="./"):
+    obj = ZipFile(zip_file, "r")
+    for item in obj.namelist():
+        obj.extract(item, out_dir)
+
+
+def zip_file_list(root_path, zip_filename, file_list=[]):
+    out_zip_file = os.path.join(root_path, zip_filename)
+    # print('debug: file_list', file_list)
+    zip_obj = ZipFile(out_zip_file, "w")
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)